### ISSUE TITLE
Test loss=0 with FIT_LARGE_SIZE_AND_PAD resize method

### ISSUE THREAD

**Issue by Goldit** [weiliu89/caffe#553]
Hi @weiliu89,

I am using FIT_LARGE_SIZE_AND_PAD resize_mode for training. The loss for training is fine, I got loss = 0 for testing.

I notice that there is a small different in resize_param for WARP and FIT_SMALL_SIZE.

'resize_param': {
                'prob': 1,
                'resize_mode': P.Resize.WARP,
                'height': resize_height,
                'width': resize_width,
                'interp_mode': [P.Resize.LINEAR],
                },

resize_param = {
        'prob': 1,
        'resize_mode': P.Resize.FIT_SMALL_SIZE,
        'height': resize_height,
        'width': resize_width,
        'height_scale': resize_height,
        'width_scale': resize_height,
        'interp_mode': [P.Resize.LINEAR],
        }

What would be the correct resize_param for training with FIT_LARGE_SIZE_AND_PAD resize mode?

Thank you and best regards,
Nhan.

**Comment by weiliu89** [comment]
That part is not fully debugged. You could use ssd.ipynb and ssd_detect.ipynb to check if anything goes wrong.
